### PR TITLE
Fixes #34652 - added git ansible module instead of command

### PR DESCRIPTION
**Pull request by snagoor**

### CODE REVIEW

File: app/views/foreman_ansible/job_templates/ansible_roles_-_install_from_git.erb [hunk 1]
```diff
@@ -23,6 +23,9 @@ model: JobTemplate
 ---
 - hosts: all
   tasks:
-    - command: git clone <%= input('git_repository') %> <%= input('location') %>
+    - git:
+        repo: <%= input('git_repository') %>
```
Comment (ezr-ondrej) [inline app/views/foreman_ansible/job_templates/ansible_roles_-_install_from_git.erb:27]: Would it be safer to add quotes? just as additional safety measure, unrelated to the actual change.
```suggestion
        repo: "<%= input('git_repository') %>"
```

**Review by ezr-ondrej — APPROVED** [review]
Make sense, thanks a bunch @snagoor ! :)
I have just one improvement idea, but I definitelly do not insist, this is ok to merge without it.

File: app/views/foreman_ansible/job_templates/ansible_roles_-_install_from_git.erb (hunk 1 above)
Comment (snagoor) [reply app/views/foreman_ansible/job_templates/ansible_roles_-_install_from_git.erb]: @ezr-ondrej agreed. Modified accordingly.

**Review by ezr-ondrej — APPROVED** [review]
Thanks @snagoor ! Nice addition :+1:

**Pull request merged**